source/install/install-tar.rst

```diff
@@ -88,6 +88,21 @@ You may also remove the Mattermost systemd unit file and the user/group created
 Frequently asked questions
 --------------------------
 
+Can I run Mattermost without a proxy?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. However, like NGINX, in place, Mattermost binds to 443 instead of 8065. The Mattermost binary requires the correct permissions to do that binding. You must activate the ``CAP_NET_BIND_SERVICE`` capability to allow the new Mattermost binary to bind to ports lower than 1024 by running the following command:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    sudo setcap cap_net_bind_service=+ep ./mattermost/bin/mattermost
+
+.. note::
+  
+  - We don't recommend deploying Mattermost on a single server unless you have fewer than 200 users.
```

Comment:
Right now we can't really make this claim and since MM Omnibus is not limited to 200 users I would remove this item.

Maybe a note that having a proxy in front on Mattermost is highly recommended and if setup/maintenance is a worry then use MM Omnibus.